lib/device/i_device.js

```diff
@@ -234,6 +234,7 @@ shaka.device.IDevice.DeviceType = {
   'MOBILE': 'MOBILE',
   'TV': 'TV',
   'VR': 'VR',
+  'APPLE_VR': 'APPLE_VR',
```

Comment:
maybe instead of adding new device type which is essentially the same as before, but with brand added, we should expose manufacturer brand in API?

Reply:
That's what I thought at first, but I don't think it's something we can easily obtain, and since we only have this one case, I've left it at that for now.

Reply:
It's not exposed API yet, so we can use it like that and see how it serves our needs.